### PR TITLE
opi testing

### PR DESCRIPTION
**Pull request by mgheorghe**
more details to be filed in as setup is catching shape and cases are contributed, the intent here is to give shape to the documentation structure.
- lab cabling diagram
- install software in the servers
- run the test cases

### CODE REVIEW

File: lab/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# OPI Lab
+
+- [Goals and Requirments](goals-and-requirements.md)
+- Setup
+  - [Bill of Materials](bom.md)
+  - [Physical Testbed Setup](README.testbed.VsSetup.md)
+  - [Server Software Setup](README.testbed.WANSetup.md)
```
Comment (glimchb) [inline lab/README.md]: last 3 are broken links

Comment (mgheorghe) [reply lab/README.md]: fixed

**Review by glimchb — APPROVED** [review]
lgtm

**Comment by mgheorghe** [comment]
those colors look too bright but should do the job

File: lab/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# OPI Lab
+
+- [Goals and Requirments](goals-and-requirements.md)
+- Setup
+  - [Bill of Materials](bom.md)
+  - [Physical Testbed Setup](physical-testbed.md)
+  - [Server Software Setup](server-setup.md)
+- [Running the Test Cases](running-the-tests.md)
```
Comment (glimchb) [inline lab/README.md:8]: you can add also this to here:
```
[Spending Guidelines](https://github.com/opiproject/opi/blob/main/Policies/spending_guidelines.md)
```

**Pull request merged**